storage/container_test.go

```diff
@@ -118,8 +146,17 @@ func TestAccContainerLifeCycle(t *testing.T) {
 	if diff := pretty.Compare(container.ExposedHeaders, updatedExposedHeaders); diff != "" {
 		t.Fatalf(fmt.Sprintf("Updated Exposed Headers diff (-got +want)\n%s", diff))
 	}
-	if container.MaxAge != _ContainerMaxAge {
-		t.Fatalf(fmt.Sprintf("Max Age do not match Wanted: %s Recieved: %s", _ContainerMaxAge, container.MaxAge))
+	if container.MaxAge != updatedMaxAge {
+		t.Fatalf(fmt.Sprintf("Max Age do not match Wanted: %d Recieved: %d", updatedMaxAge, container.MaxAge))
+	}
+	if container.QuotaBytes != updatedQuotaBytes {
+		t.Fatalf(fmt.Sprintf("Quota Bytes do not match Wanted: %d Recieved: %d", updatedQuotaBytes, container.QuotaBytes))
+	}
+	if container.QuotaCount != updatedQuotaCount {
+		t.Fatalf(fmt.Sprintf("Quota Count do not match Wanted: %d Recieved: %d", updatedQuotaCount, container.QuotaCount))
+	}
+	if diff := pretty.Compare(container.CustomMetadata, updatedCustomMetaData); diff != "" {
+		t.Fatalf(fmt.Sprintf("CustomMetadata diff (-got +want)\n%s", diff))
```

Comment:
Got an error from the test here 
```
	container_test.go:159: CustomMetadata diff (-got +want)
		 {
		  Abc-Def: "123",
		  Bar: "foo",
		- Foo: "bar",
		 }
```

Reply:
Fixed. Update to support explicit removal of custom metadata headers, so `Foo` now gets removed.